### ISSUE TITLE
Cannot use the plugin on iOS, missing dependency

### ISSUE THREAD

**Issue by sneko** [viveapp/capacitor-plugin-freshchat#2]
@EKashpersky @rroukens ,

I worked around #1 by using the Github repository URL instead of a NPM package, which allows me to do `npx cap sync`...

The issue is when trying to build for iOS it says `FreshchatSDK` pod is missing. So I tried to add it manually into `Podfile`:
```
pod 'FreshchatSDK'
```

So a `pod install` will work but when building I get multiple errors:
```
Command CompileSwiftSources failed with a nonzero exit code

Command EmitSwiftModule failed with a nonzero exit code

Command CompileSwiftSources failed with a nonzero exit code

...
```

Do you know what's missing to make things work?

Thank you,

**Comment by EKashpersky** [comment]
@sneko hi

I've updated documentation, look for initialisation steps for iOS.

Cheers

**Comment by sneko** [comment]
@EKashpersky do I need to do this with or without adding manually:
```
pod 'FreshchatSDK'
```
?

I tried without (as specified in your updated documentation) but ...
<img width="737" alt="image" src="https://user-images.githubusercontent.com/8195958/188671733-3b33ace7-71cf-4440-97c6-5444e644e1f0.png">
<img width="397" alt="image" src="https://user-images.githubusercontent.com/8195958/188672492-7dcb3b38-d22a-44e9-8222-dd34bf6f2e36.png">


There is nothing to add... When searching for `FreshchatSDK.xcfrmework`.

Did you try it from scratch? Curious to have screenshot if no step is missing in the how-to.

Thank you,


**EDIT: Ok, I'm able to link the library only if I set back manually the pod as I mentioned above, which is missing in your explanations (or in the auto-adding when doing `npx cap sync`). Able to build right now, will try on an iPhone soon ;)**
